readme.txt

```diff
@@ -80,6 +80,7 @@ Added: Dashboard support widget
 Added: Tax to COD surcharge and Age verification fee
 Added: Stockie theme support
 Added: Calculation of exceeding the Free shipping limit without discount from coupons
+Added: Calculation of exceeding the Free shipping limit with taxes
```

Comment:
"S daněmi" nevystihuje to, že to nově respektuje nastavení zadávání cen produktů buď včetně daně nebo bez daně

Reply:
Ne, nerespektuje, je to na tomhle nastaveni nezavisle. Viz dikuze u pozadavku.

Reply:
Ano, tak to myslím, napsat, že to je na tomto nastavení nezávislé.

Reply:
Added: Calculation of exceeding the Free shipping limit with taxes (if applicable)

?

Reply:
To zase říká něco jiného. Dal bych třeba "Free shipping limit calculation independent of tax or tax-free pricing settings"